core/test/common/MultiRole.js

```diff
@@ -191,4 +192,34 @@ contract("MultiRole", function(accounts) {
     assert.isFalse(await multiRole.holdsRole("2", account3));
     assert(await didContractThrow(multiRole.revertIfNotHoldingRole("2", { from: account3 })));
   });
+
+  it("Events are emitted", async function() {
+    const multiRole = await MultiRoleTest.new();
+    await multiRole.createExclusiveRole("1", "1", account1);
+    await multiRole.createSharedRole("2", "1", [account2, account3]);
+
+    // Add shared member.
+    const addSharedResult = await multiRole.addMember("2", account4, { from: account1 });
+    truffleAssert.eventEmitted(addSharedResult, "AddedSharedMember", ev => {
+      return ev.roleId.toString() == "2", ev.newMember == account4, ev.manager == account1;
+    });
+
+    // Remove shared member.
+    const removeSharedResult = await multiRole.removeMember("2", account4, { from: account1 });
+    truffleAssert.eventEmitted(removeSharedResult, "RemovedSharedMember", ev => {
+      return ev.roleId.toString() == "2", ev.oldMember == account4, ev.manager == account1;
```

Comment:
What does the comma do here, as opposed to `&&`?

Reply:
Weird, that was produced via linting. I'm testing its behavior locally now

Reply:
Ah, this was not the linter's fault, it was my fault I accidentally replaced all &&'s with commas. Good catch @ptare